README.md

```diff
@@ -1,31 +1,70 @@
 [RethinkDB](http://www.rethinkdb.com)
 =================
 
-RethinkDB is an open-source distributed database. It has an intuitive
-query language, automatically parallelized queries, and simple
-administration.
+RethinkDB is the first open-source scalable database built for realtime applications. It exposes a new database access model -- instead of polling for changes, the developer can tell the database to continuously push updated query results to applications in realtime. RethinkDB allows developers to build scalable realtime apps in a fraction of the time with less effort.
 
-To get started, check out [rethinkdb.com](http://rethinkdb.com)!
+To learn more, check out [rethinkdb.com](http://rethinkdb.com).
 
-Quick start
+Not sure what types of projects RethinkDB can help you build? Here are a few examples:
+
+* Build a [realtime liveblog](http://rethinkdb.com/blog/rethinkdb-pubnub/) with RethinkDB and PubNub
+* Create a [collaborative photo sharing whiteboard](https://www.youtube.com/watch?v=pdPRp3UxL_s)
+* Build an [IRC bot in Go](http://rethinkdb.com/blog/go-irc-bot/) with RethinkDB changefeeds
+* Look at [cats on Instagram in realtime](http://rethinkdb.com/blog/cats-of-instagram/)
+* Watch [how Platzi uses RethinkDB](https://www.youtube.com/watch?v=Nb_UzRYDB40) to educate
+
+
+Quickstart
 -----------
 
-For a thirty second quickstart, look at http://www.rethinkdb.com/docs/guides/quickstart
+For a thirty-second RethinkDB quickstart, check out  [rethinkdb.com/docs/quickstart](http://www.rethinkdb.com/docs/quickstart).
+
+Or, get started right away with our ten-minute guide in these languages:
+
+* [**JavaScript**](http://rethinkdb.com/docs/guide/javascript/)
+* [**Python**](http://rethinkdb.com/docs/guide/python/)
+* [**Ruby**](http://rethinkdb.com/docs/guide/ruby/)
 
-Get help
+Besides our three official drivers, we also have many [community drivers](http://rethinkdb.com/docs/install-drivers/) supported by the RethinkDB community. Here's a few:
+
+* **C#/.NET:** [rethinkdb-net](https://github.com/mfenniak/rethinkdb-net)
+* **Clojure:** [clj-rethinkdb](https://github.com/apa512/clj-rethinkdb)
+* **Go** [GoRethink](https://github.com/dancannon/gorethink)
+* **Haskell:** [haskeel-rethinkdb](https://github.com/atnnn/haskell-rethinkdb)
+* **PHP:** [php-rql](https://github.com/danielmewes/php-rql) - [@danielmewes](https://github.com/danielmewes)
+* **Scala** [rethink-scala](https://github.com/kclay/rethink-scala) - [@kclay](https://github.com/kclay)
+
+
+Need help?
 --------
 
-Check out www.rethinkdb.com/community
+If you need help, a great place to start is  [rethinkdb.com/community](www.rethinkdb.com/community). Here you can find out how to ask us questions, reach out to us, or [report an issue](https://github.com/rethinkdb/rethinkdb/issues). As well, find all the places we frequent online and which conference or meetup you might be able to meet us next.
```

Comment:
The sentence that starts, “As well, find...”. I think you want something like, “You’ll also be able to see all the places we frequent online, as well as which conference or meetup we'll be at next." The whole change isn't necessary, but the "As well," fragmented clause one is.